### PR TITLE
Security crowbars but with less shitcode

### PR DESCRIPTION
**Pull request by Nobelium-XVIII**
## About The Pull Request

they actually have the rest of their gear now

## Why It's Good For The Game

see above

## Changelog
:cl:
fix: makes it so spawning the crowbar doesn't delete everything else
/:cl:

### CODE REVIEW

File: fulp_modules/ZFulpcode/modules/jobs/inventories.dm [hunk 1]
```diff
@@ -1,13 +1,13 @@
 /// Security (Original by Surrealistik) ///
 
 /datum/outfit/job/security
-	backpack_contents = list() //They start out with a crowbar!
-	box = /obj/item/storage/box/survival/security/improved
+	backpack_contents = list(/obj/item/melee/baton/loaded=1) //we want to keep the stuff they had before
```
Comment (GuillaumePrata) [inline fulp_modules/ZFulpcode/modules/jobs/inventories.dm:4]: backpack_contents = list(/obj/item/melee/baton/loaded=1) //we want to keep the stuff they had before
```suggestion

```
This line will overwrite any new item added to backpacks and will need to be updated, it also simply doesn't need to exist.

File: fulp_modules/ZFulpcode/modules/jobs/inventories.dm [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+	box = /obj/item/storage/box/survival/security/improved //then put the crowbar into their survival box
 
 /datum/outfit/job/warden
-	backpack_contents = list() //They start out with a crowbar!
+	backpack_contents = list(/obj/item/melee/baton/loaded=1)
```
Comment (GuillaumePrata) [inline fulp_modules/ZFulpcode/modules/jobs/inventories.dm:8]: Same here

File: fulp_modules/ZFulpcode/modules/jobs/inventories.dm [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 	box = /obj/item/storage/box/survival/security/improved
 
 /datum/outfit/job/hos
-	backpack_contents = list() //They start out with a crowbar!
+	backpack_contents = list(/obj/item/melee/baton/loaded=1, /obj/item/modular_computer/tablet/preset/advanced/command=1)
```
Comment (GuillaumePrata) [inline fulp_modules/ZFulpcode/modules/jobs/inventories.dm:12]: And here

**Comment by JohnFulpWillard** [comment]
Verified label please

**Pull request merged**